tests/integration/cairo1/parser_test.py

```diff
@@ -10,25 +10,7 @@
 
 @pytest.fixture(name="test_suite_json")
 def test_suite_json_fixture(datadir: Path) -> str:
-    """
-    Cairo source code of the tested fixture
-    -----------------------------------------
-    fn test_cheatcode_caller() {
-       roll(1, 2)
-    }
-
-    fn test_cheatcode_caller_twice() {
-       roll(1, 2);
-       roll(1, 2)
-    }
-
-    fn test_cheatcode_caller_three() {
-       roll(1, 2);
-       roll(1, 2);
-       roll(1, 2)
-    }
-    -----------------------------------------
-    """
+    # Cairo source code of the tested fixture - CAIRO_ROLL_TEST
     with open(datadir / "compiled_test_suite.json", "r") as file:
```

Comment:
Consider renaming  "compiled_test_suite.json" and moving it next to the source code.

Reply:
I don't think the location of this file is that important, the source code can be easily found and it's just additional info here.